src/api/User/types.ts

```diff
@@ -7,11 +7,20 @@
 import {GenericObject, IDataSourceParams, PersistedItem} from '../commonTypes';
 import {AccessRight, Action, AnyAction} from '../AccessRight';
 import {IGuestPreferencesConfig} from '../Config';
+import {CaseStatus, FullCaseListSortBy} from '../Alert';
+
+export interface IUnifiedCaseListPreferences {
+  alertIdsFilter?: number[];
+  caseStatusesFilter: CaseStatus[];
```

Comment:
caseStatusesFilter should be optional

Reply:
ahh sorry, no changes required here

Reply:
why no changes required? I thought we said it would be optional?